front/src/reducers/osrdconf/helpers.ts

```diff
@@ -142,3 +145,47 @@ export const updateDestinationPathStep = (
   destination: Partial<PathStep> | null,
   replaceCompletely: boolean = false
 ) => updatePathStepAtIndex(pathSteps, pathSteps.length - 1, destination, replaceCompletely);
+
+/**
+ * modifies the array statePathSteps in place
```

Comment:
Maybe to clarify why we chose to mutate the parameter

```suggestion
 * Modifies the array statePathSteps in place in the reducer
```

Reply:
Should we add a test for it ?

Reply:
sure, I opened a ticket https://github.com/OpenRailAssociation/osrd/issues/8836

Reply:
changed the tsdoc :)